packages.tsv

```diff
@@ -9,6 +9,8 @@ alpine	jq	1	test $(/usr/bin/jq -M -n -c "{a: (4+3)}") = "{\"a\":7}"
 alpine	mtr	1	/usr/sbin/mtr -v
 conda	m4	1.4.17--0	test $(echo "define(a, bbb)a" | m4) = "bbb"
 conda	samtools	1.3--1	samtools --version
+conda	samtools	1.2--2	samtools --version
```

Comment:
I can drop by when it's most convenient for you. Yes, let's do that. I'll add more polish here and there to make it presentable :wink: